lib/cea/cea608_memory.js

```diff
@@ -0,0 +1,513 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.cea.Cea608Memory');
+goog.provide('shaka.cea.Cea608Char');
+
+/**
+ * CEA-608 captions memory/buffer.
+ */
+shaka.cea.Cea608Memory = class {
+  /**
+   * @param {!shaka.cea.CeaDecoder} decoder CEA-608 decoder.
+   * @param {!number} fieldNum Field number.
+   * @param {!number} chanNum Channel number.
+   * @param {!boolean} textmode Indicates whether text mode is on/off.
```

Comment:
I don't understand what this does.  It doesn't seem to do anything, but if it's part of a plan for future enhancements, the name and description don't give much indication of what it's for.  At least, for someone like me who doesn't know CEA 608 at any significant level of detail.

Reply:
I agree. As explained earlier, it doesn't seem like we will ever have plans to actually emit text mode, so I've removed the field from the memory buffer. However, text mode is still going to be handled as a no-op on the data channel level.